corehq/apps/hqwebapp/utils/bootstrap/reports/stats.py

```diff
@@ -0,0 +1,89 @@
+import phonelog.reports as phonelog
+
+from corehq.apps.accounting import interface as accounting
+from corehq.apps.enterprise.interface import EnterpriseSMSBillablesReport
+from corehq.apps.fixtures import interface as fixtures
+from corehq.apps.geospatial import reports as geospatial
+from corehq.apps.hqadmin import reports as hqadmin
+from corehq.apps.linked_domain.views import DomainLinkHistoryReport
+from corehq.apps.reports import commtrack
+from corehq.apps.reports.standard import (
+    deployments,
+    inspect,
+    monitoring,
+    sms,
+)
+from corehq.apps.reports.standard.cases.basic import CaseListReport
+from corehq.apps.reports.standard.cases.case_list_explorer import CaseListExplorer
+from corehq.apps.reports.standard.cases.duplicate_cases import DuplicateCasesExplorer
+from corehq.apps.reports.standard.forms import reports as receiverwrapper
+from corehq.apps.reports.standard.project_health import ProjectHealthDashboard
+from corehq.apps.reports.standard.users.reports import UserHistoryReport
+from corehq.apps.sso.views.accounting_admin import IdentityProviderInterface
+from corehq.motech.generic_inbound.reports import ApiRequestLogReport
+from corehq.motech.repeaters.views import DomainForwardingRepeatRecords
+
+
+ALL_REPORTS = [
```

Comment:
I'm not too concerned about enforcing that this list is kept up-to-date as new reports are added. I think it will be clear to anyone who goes to a report missing in this list to add it here if they migrate the report and try to run `complete_bootstrap5_report TheirReportView`.